dist/api_data.py

```diff
@@ -574,6 +574,11 @@ def __ge__(self, other):
         Config('hashsize', '1024', r'''
             number of buckets in the hashtable that keeps track of objects''',
             min='64', max='1048576'),
+        Config('pinned', '', r'''
+            List of "table:" URIs exempt from cache eviction. Capacity config overrides this,
+            not fully retaining larger tables exceeding capacity. Table names can appear
```

Comment:
> not fully retaining larger tables exceeding capacity.
NP: maybe a reword - "tables exceeding capacity will not be fully retained."

Reply:
Updated in [7ee8781](https://github.com/wiredtiger/wiredtiger/pull/9415/commits/7ee8781ffa2804d3da4a3e7154fc669554d67558)